lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
I think that's correct. If Skylight can track two environments by default then there's not really any good reason why you wouldn't want to go with that.